config/shared/errors/errors.go

```diff
@@ -81,6 +81,9 @@ var (
 	ErrDuplicateLabels           = errors.New("cannot use the same partition label twice")
 	ErrInvalidProxy              = errors.New("proxies must be http(s)")
 	ErrInsecureProxy             = errors.New("insecure plaintext HTTP proxy specified for HTTPS resources")
+	ErrFileConflictsSystemd      = errors.New("file path conflicts with systemd unit or dropin")
+	ErrLinkConflictsSystemd      = errors.New("link path conflicts with systemd unit or dropin")
+	ErrDirConflictsSystemd       = errors.New("directory path conflicts with systemd unit or dropin")
```

Comment:
In other cases we spell out `Directory` in error names.  We should probably do that here even though `gofmt` will re-indent everything.  Or alternatively, there's not really a reason to have three separate errors here; we could just make `ErrPathConflictsSystemd`.